webapp/channels/src/components/admin_console/license_settings/renew_license_card/renew_license_card.tsx

```diff
@@ -15,7 +15,7 @@ import {getRemainingDaysFromFutureTimestamp} from 'utils/utils';
 
 import './renew_license_card.scss';
 
-export interface RenewLicenseCardProps {
```

Comment:
Odd that this is safe to remove since I've definitely run into other places where having props be an unexported `interface` (instead of an exported `interface` or a `type`) would cause VS Code to complain, but it doesn't seem to because of this change. It doesn't seem to affect actually type checking or building at all though, so it's mostly just annoying if it does happen though.

I guess it must only happen for components that use React Redux's connect since I'm not seeing any errors when I tested this locally. Since we're switching many things away from `connect` anyway, there's nothing to change here